dkg/dkg.go

```diff
@@ -142,29 +142,38 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return errors.New("unsupported dkg algorithm")
 	}
 
-	log.Info(ctx, "Successfully completed DKG ceremony, writing output")
-
+	log.Info(ctx, "Writing keyshares to disk")
```

Comment:
why is this info and the rest debug?

Reply:
good catch, it's one of the debug guys disguising as info. i'll change it